src/Sidekick.Presentation.Blazor.Electron/Startup.cs

```diff
@@ -111,24 +111,6 @@ public void ConfigureServices(IServiceCollection services)
             Task.Run(async () =>
             {
                 trayProvider.Initialize();
-                var browserWindow = await ElectronNET.API.Electron.WindowManager.CreateWindowAsync(new BrowserWindowOptions
-                {
-                    Width = 1,
-                    Height = 1,
-                    Frame = false,
-                    Show = true,
-                    Transparent = true,
-                    Fullscreenable = false,
-                    Minimizable = false,
-                    Maximizable = false,
-                    SkipTaskbar = true,
-                    WebPreferences = new WebPreferences()
-                    {
-                        NodeIntegration = false,
-                    }
-                });
-                await Task.Delay(50);
-                browserWindow.Close();
```

Comment:
What I found to remove the splashscreen is we need to create a window with ElectronNET. There wiil be no guarantee that the initialization view will open because we have a setting to remove the initialization window. So I think we need to keep this.

Reply:
You can always keep it and put Show to `false` in `BrowserWindowOptions`.

Reply:
I shouldn't have committed this, I was trying to track down why the About Window was always opening on load. I'll revert this.

Reply:
Restored in latest commit